Misc/NEWS.d/next/Core and Builtins/2019-03-20-00-38-35.bpo-36375.d178R0.rst

```diff
@@ -0,0 +1,8 @@
+PEP 499 implemented: the "python -m module_name" command line option now binds the imported module
+to both ``sys.modules['__main__'`` and additionally to ``sys.modules['module_name']``.

```

Comment:
``sys.modules['__main__']``?

Reply:
Nice catch, fix applied, thank you! - Cameron